code/optimize/build.gradle

```diff
@@ -205,8 +205,7 @@ task platformFunctionalTestJacocoReport(type: JacocoReport, dependsOn: "createPh
 dependencies {
     implementation 'androidx.appcompat:appcompat:1.5.1'
```

Comment:
app compat brings in a lot of dependencies, let's use any specific libraries we need here e.g. annotation 
```
implementation 'androidx.annotation:annotation:1.0.0'
```
This will help resolve the multidex issue.